api/src/database/index.ts

```diff
@@ -127,3 +129,17 @@ export async function isInstalled(): Promise<boolean> {
 	// exists when using the installer CLI.
 	return await inspector.hasTable('directus_collections');
 }
+
+export const validateMigrations = async (): Promise<boolean> => {
```

Comment:
Lets use the same code style as elsewhere:

```suggestion
export async function validateMigrations() {
```